Doc/library/pprint.rst

```diff
@@ -61,6 +61,9 @@ Functions
    :class:`PrettyPrinter` constructor and their meanings are as
    described in its documentation below.
 
+   Note that *sort_dicts* default is ``True`` and you might want to use
+   :func:`~pprint.pp` instead.::
```

Comment:
The example below still uses `pprint`.

Reply:
Then we should probably move the example to the top, but it would also make sense to provide full description there (in `pp`).

Reply:
It might even make sense to combine this change with your earlier PR (#116085, but a bit improved)